app/assets/javascripts/models/page/search_page.js.coffee

```diff
@@ -50,7 +54,11 @@ ns.SearchPage = do (ko
     $('.launch-customize-modal').click ->
       $('#customizeDataModal').modal('show')
 
-    preferences.ready.done(-> initModal()) if !window.edscportal
+    reverb = ["http://echo-reverb-rails.dev", "https://testbed.echo.nasa.gov", "https://api-test.echo.nasa.gov", "https://testbed.echo.nasa.gov", "https://reverb.echo.nasa.gov"]
```

Comment:
Set the reverb URLs in config file, not here. Also leave the option to ops so that they can inject the URL from bamboo

Reply:
Mmmmmm...but wouldn't that interfere with the 'hat' philosophy?  Storing the URL's in a config file seems - embedded.  We won't need them any longer after this migration, so why keep them stored so deeply?

Reply:
What I am worried about is between now and early next year, there will be changes inserted among the lines that you did today which, when we do git revert or the manual eyeball revert, will have some risks of modifying existing features and/or not to be removed completely. Add new values in services makes the changes scattered, yes, but it won't hurt us when we do the revert.